lib/Backend/GlobOpt.cpp

```diff
@@ -307,7 +308,12 @@ GlobOpt::Optimize()
         // Keep track of the last symbol for which we're guaranteed to have data.
         this->maxInitialSymID = this->func->m_symTable->GetMaxSymID();
         this->BackwardPass(Js::BackwardPhase);
+
+        // changedSymsAfterIncBailoutCandidate helps track building incremental bailout in ForwardPass
+        this->changedSymsAfterIncBailoutCandidate = JitAnew(alloc, BVSparse<JitArenaAllocator>, alloc);
```

Comment:
Can you initialize this bit vector along with the other members of GlobOpt in ForwardPass() ?